internal/core/capi.cpp

```diff
@@ -237,66 +285,24 @@ int metadata(
     }
 }
 
-int horizon_buffer_offsets(
+int fetch_subvolume(
```

Comment:
1. Wrong comment place - your docs are for fetch_subvolume from cppapi.hpp, yet your comment is for capi.hpp
2. This function doesn't interpolate and resample. This function only loads the data, which is represented by name fetch.
3. Do you feel like this describing parameters gives you anything here?
You write: handle is a data (what else can it be?). subvolume is a 3d data  (what else can it be). interpolation is interpolation. from is start and to is end.
So I feel like describing parameters here becomes redundant the moment one is more-or-less familiar with the code. And having simple parameters described in every function (because if we add it here, why shouldn't we add it everywhere else?) just hurts readability as you would have a lot of not-so-much interesting information flying around.

If you want to I can add to whichever (C or C++) fetch subvolume docs like
"Fetch data of the provided subvolume in a range [from, to) on the horizontal grid"

Reply:
1. Indeed you are right. No idea how it happened. 
2. Agreed that fetch implies that the function is only reading data. On the other hand, interpolation parameter suggest otherwise. My understanding of the code:

```  
    handle.read_samples(
        subvolume.data(from),
        size,
        samples.get(),
        nsamples,
        interpolation
    );
```

is that the data is interpolated and resampled.  Please explain what the interpolation parameter is used for if the function do not do the interpolation?

3. I thought that this was a neat way to add some clarity to none author readers. The ´from´ and ´to´ are problematic variable names. As they are the start and end index of a grid (x, y) cordinate listed in a predetermined order defined outside this function. This grid is in the next level used to define the volume described by the sub-volume. Code wise all this is fine by me, but it is a tall order to ask the reader to dig through the code and figure it out. I know ;-). The handle can also be the target buffer, this is not uniquely defined. 

4. If I misunderstood the purpose of the function, then that should be an argument in favour of adding a function description.

Reply:
2 - ok, you are right. It is just that resampling and interpolation here is not that interesting. This is resampling/interpolation in the horizontal surface, I thought you meant vertical interpolation which we explicitly do in attributes. This interpolation and resampling is done by openvds.

I suggest that as we disagree on the docs in general, we discuss it later and take it as a separate issue not related to this PR (as it is not related to this PR)

Reply:
Please do provide some aide to the reader on the parameters ´from´ and ´to´ these are critical and non trivial.